manifests/params.pp

```diff
@@ -70,21 +78,25 @@
         # needed for mysqlproxy
         'glib2-devel',
         # needed to build zookeeper
-        #'java-1.7.0-openjdk',
+        'java-1.8.0-openjdk',
```

Comment:
Essentially -- RedHat is fairly aggressive about getting security patched versions out the door.